rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ProjectSpecTest.java

```diff
@@ -483,4 +492,14 @@ public void should_document_create_project() throws Exception {
                                 fieldWithPath("_embedded.createdBy").description("The user who created this project")
                         )));
     }
+
+	@Test
+	public void should_document_create_releases() throws Exception {
+		List<String> releaseIds = Arrays.asList("3765276512", "5578999", "3765276513");
+
+		String accessToken = TestHelper.getAccessToken(mockMvc, testUserId, testUserPassword);
+		this.mockMvc.perform(post("/api/projects/" + project.getId() + "/releases").contentType(MediaTypes.HAL_JSON)
+				.content(this.objectMapper.writeValueAsString(releaseIds))
+				.header("Authorization", "Bearer " + accessToken)).andExpect(status().isCreated());
```

Comment:
Shouldn't the `.andDo(this.documentationHandler.document(...` part be coming after `.andExpect()` so that all the input request params and response params get explained?

Reply:
This API only accepts an array of release ids as request body, there won't be any other params/fields as such and returns only http status and header information as response, there won't be any other response params either. So I think no need to add the documentation for the same. However to make it clearer for the user, I have updated the description section mentioning about what should be the request body. Please let me know, if it is ok.